### ISSUE TITLE
Creating a GitHub Action that tweets new Awesome Entries

### ISSUE THREAD

**Issue by Ly0n** [sindresorhus/awesome#1920]
Hey Awesome people,

Problem:
I love awesome lists but they do have one essential problem. It is very hard to follow new projects on awesome lists.
It would be nice to create a newsfeed for readers to stay informed about new entries. 

Solution.
We created a simple GitHub Action that creates tweets based on the commit messages. Contributors can define there own custom tweet message by defining the Pull Request title in a corresponding way. 

The script checks if the last commit message on the main branch contains `https` and creates a tweet based on the corresponding line.
https://github.com/protontypes/awesome-twitter-action

1. Add the Github Action Script to .github/workflows/tweet.yml
2. Create a Twitter Developer Account.
3. Add the Login data to your project secrets.
4. Activate your Github Actions by pressing the Activate GitHub Action for this Projects button on the Action Tab of your project.
5. Create a Commit Message with an URL. Since the title of a pull request is included in the commit message, contributors can create user-specific Twitter messages.

It would be nice to get your feedback. The action is already integrated here: 
https://github.com/protontypes/awesome-robotic-tooling

Do you see any better integration that might be more intuitive and fit better in the Awesome list workflow? At the moment we assume that every commit is always a new project entry.

**Comment by MsPinkieBoo7872** [comment]
I am looking for a first project would this be a good fit for a beginner.

**Comment by Jogai** [comment]
New entries are already tweeted

**Comment by JohnNewbold** [comment]
Use the google earth app to stretch size of pictures material

**Comment by sammluc** [comment]
Follow this person margo2016tepa
